core/src/main/java/org/mule/runtime/core/api/processor/strategy/ProcessingStrategy.java

```diff
@@ -57,4 +59,19 @@ default boolean isSynchronous() {
     return false;
   }
 
+  /**
+   * Checks whether backpressure will be fired for a new accepted {@link org.mule.runtime.api.event.Event} to be processed.
+   *
+   * @throws RejectedExecutionException
+   */
+  default void checkBackpressureAccepting(CoreEvent event) throws RejectedExecutionException {}
+
+  /**
+   * Checks whether backpressure will be fired for a new emitted {@link org.mule.runtime.api.event.Event} to be processed.
+   *
+   * @throws RejectedExecutionException
```

Comment:
doesn't this method return false is it should trigger backpressure?

Reply:
You are right. Just corrected javadoc.